### DISCUSSION TITLE
variable to remove prediction?

### DISCUSSION THREAD

**Discussion by JoeyJoe6** [djdd87/SynoAI#215] [Q&A]
Since I have not had the time to properly train the AI, is there a variable I can use to switch off what the AI classifies the target? I would rather just have the outlined box and leave out the cat, dog, person, etc..

**Comment by de1et3d** [comment]
"DrawMode": "All"
"FontColor": "#00FFFFFF"
"TextBoxColor": "#00FFFFFF"

**Reply by djdd87** [reply]
Yep, setting them to transparent should work fine for the moment, but I can add a config to just disable labels.
